src/pages/graphql/payment-methods/payment-services-checkout.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Payment Services during a checkout authorization
+description: Learn how Adobe Commerce uses GraphQL to process orders for the Payment Services payment method during a typical checkout authorization.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment services during a checkout authorization
+
+[Payment Services for Adobe Commerce and Magento Open Source](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/guide-overview.html) is your turnkey self-service solution, including sandbox testing and a simple setup, for providing robust and secure payment processing for your Commerce websites.
+
+These steps describe the flow of requests and responses required to complete a typical [checkout](../tutorials/checkout/index.md) authorization with Payment Services enabled.
+
+## Checkout workflow
+
+![Payment Services sequence diagram](../../_images/graphql/payment-services.svg)
+
+1. Run `getPaymentConfig` to fetch the payment configuration to retrieve and render details about PayPal components, such as hosted fields, smart buttons, and Apple Pay.
+
+1. Commerce returns payment configuration information.
+
+1. Run `setPaymentMethodOnCart` to [set the payment method](../tutorials/checkout/set-payment-method.md).
+
+1. Adobe Commerce returns a `Cart` object.
+
+1. Run `createPaymentOrder` to begin the authorization process.
+
+1. Commerce forwards the request to PayPal.
+
+1. PayPal returns an `id` value.
+
+1. Adobe Commerce generates a `order_id` and forwards the value in the `mp_order_id` field and the PayPal response in the `id` field.
+
+1. (Optional) If hosted fields and the Signifyd integration are enabled, run `getPaymentOrder`.
+
+1.  (Optional) Adobe Commerce returns details about the payment order.
+
+1.  Run `placeOrder`.
+
+1.  Commerce sends an authorization request to PayPal.
+
+1.  PayPal returns the result to Commerce.
+
+1.  Commerce creates an order.
+
+## `setPaymentMethodOnCartInput` object
+
+When you set the payment method to Payment Services in the [`setPaymentMethodOnCart`](../schema/cart/mutations/set-payment-method.md) mutation, the `setPaymentMethodOnCartInput` object must contain specific `Payment Services` attributes for the supported payment methods.
+
+### Payment Services object
+
+The `Payment Services` object must contain the following attributes:
+
```

Comment:
You can say all 3 or 4 payment methods contain `payments_order_id`, `paypal_order_id` and  `payment_source` (all presented in a table). Then mention vault contains `public_hash` and hosted fields contain these

is_active_payment_token_enabler
cardBin
cardExpiryMonth
cardExpiryYear
cardLast4
holderName

I suppose you could put them all in a single table and add a column that indicates which method uses each attribute.